biogtr/models/embedding.py

```diff
@@ -34,42 +139,15 @@ def _torch_int_div(
         """
         return torch.div(tensor1, tensor2, rounding_mode="floor")
 
-    def _sine_box_embedding(
-        self,
-        boxes,
-        features: int = 512,
-        temperature: int = 10000,
-        scale: float = None,
-        normalize: bool = False,
-        **kwargs,
-    ) -> torch.Tensor:
+    def _sine_box_embedding(self, boxes: torch.Tensor) -> torch.Tensor:
         """Compute sine positional embeddings for boxes using given parameters.
 
         Args:
             boxes: the input boxes.
```

Comment:
Document the expected shapes of `boxes` since there's some shape handling below (e.g., for rank 2 inputs)